### PR TITLE
Send invitaion with user create

### PR DESCRIPTION
**Pull request by zumerahmadbajwa**

### CODE REVIEW

File: app/controllers/invitations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    @user.password = SecureRandom.base64(10)
+    UserMailer.send_invitation(@user, @user.password).deliver_later if @user.save
+    redirect_to admin_users_path
```
Comment (Sohair63) [inline app/controllers/invitations_controller.rb:15]: what if the invitation do not save?

File: app/controllers/invitations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class InvitationsController < ApplicationController
+  # skip_before_action :confirmable, only: :create
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    @user.password = SecureRandom.base64(10)
```
Comment (Sohair63) [inline app/controllers/invitations_controller.rb:13]: use alphanumeric rather than base64

File: app/controllers/invitations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Invitation Controller
+class InvitationsController < ApplicationController
+  # skip_before_action :confirmable, only: :create
```
Comment (Sohair63) [inline app/controllers/invitations_controller.rb:5]: remove this if we are not using it.

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-  <% end %>
+  <div class="text-light">
+    <% if user_signed_in? %>
+        <%= current_user.email %>
+        <%= link_to "Log_out", destroy_user_session_path, method: :delete %>
+        <%= link_to "Account Settings", edit_user_registration_path %>
+        <% if current_user.admin? %>
+          <%= link_to "User's list", admin_users_path %>
+        <% end %>
+    <% else %>
```
Comment (Sohair63) [inline app/views/home/index.html.erb:11]: indentation issue

File: app/views/admin/users/index.html.erb [hunk 5]
```diff
@@ -34,8 +34,11 @@
           <% end %>
         </tbody>
       </table>
-      <%= paginate @users, theme: 'bootstrap-5', pagination_class: "pagination-sm flex-wrap justify-content-center",
-      nav_class: "d-inline-block" %><br>
+      <div class="float-end btn btn-warning text-wrap">
+          <%= link_to 'Create', new_invitation_path%>
```
Comment (Sohair63) [inline app/views/admin/users/index.html.erb:38]: indentation

File: app/controllers/invitations_controller.rb (hunk 2 above)
Comment (zumerahmadbajwa) [reply app/controllers/invitations_controller.rb:13]: Because of some password validation like: special char, numeric value etc, I used it

**Pull request merged**